saito-rust/configs/issuance.template

```diff
@@ -0,0 +1 @@
+100000000	03145c7e7644ab277482ba8801a515b8f1b62bcd7e4834a33258f438cd7e223849	VipOutput
```

Comment:
Is VipOutput a type or a label? if it's label, should be changed to 'Spammer'

Reply:
It is a type, There is no Spammer type in transactions.rs